lib/mayaUsd/ufe/UsdHierarchy.cpp

```diff
@@ -201,35 +202,35 @@ Ufe::InsertChildCommand::Ptr UsdHierarchy::insertChildCmd(
 }
 
 Ufe::SceneItem::Ptr UsdHierarchy::insertChild(
-        const Ufe::SceneItem::Ptr& ,
-        const Ufe::SceneItem::Ptr& 
+        const Ufe::SceneItem::Ptr& child,
+        const Ufe::SceneItem::Ptr& pos
 )
 {
-    // Should be possible to implement trivially when support for returning the
-    // result of the parent command (MAYA-105278) is implemented.  For now,
-    // Ufe::Hierarchy::insertChildCmd() returns a base class
-    // Ufe::UndoableCommand::Ptr object, from which we can't retrieve the added
-    // child.  PPT, 13-Jul-2020.
+    #ifdef UFE_V2_FEATURES_AVAILABLE
+    auto insertChildCommand = insertChildCmd(child, pos);
+    return insertChildCommand->insertedChild();
+    #else
     return nullptr;
+    #endif
```

Comment:
This change is not related to this PR but decided to add this change anyways since UFE now support this.

Reply:
Sorry about the whitespace changes here, Please enable the "Hide whitespace changes" to ignore them. My text editor doesn't like tabs :).